pyhap/accessory_driver.py

```diff
@@ -210,7 +215,7 @@ def __init__(self, *, address=None, port=51234,
         self.topics = {}  # topic: set of (address, port) of subscribed clients
         self.topic_lock = threading.Lock()  # for exclusive access to the topics
         self.loader = loader or Loader()
-        self.aio_stop_event = asyncio.Event(loop=self.loop)
+        self.aio_stop_event = asyncio.Event()
```

Comment:
This change is breaking Python 3.5 support. It's not necessary, just a drive by cleanup from when I saw warnings printed during tests. I will revert it.